apps/frontend-pwa/src/components/participant/groups/GroupJoinBlock.tsx

```
@@ -29,6 +30,13 @@ function GroupJoinBlock({
         buttonMode={false}
         title={t('pwa.courses.joinGroup')}
         icon={faPeopleGroup}
+        validationSchema={Yup.object().shape({
+          value: Yup.string()
+            .required(t('pwa.groups.pinRequired'))
+            .test('is-numeric', t('pwa.groups.pinNumeric'), (value) => {
+              return !isNaN(Number(value)) && value.length === 6
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)